pymine_net/types/player.py

```diff
@@ -10,13 +11,11 @@
 from pymine_net.types.vector import Rotation, Vector3
 
 
+@dataclass(slots=True)
```

Comment:
The slots kwarg is is a 3.10+ feature iirc, please revert back to using a regular class.